src/PrestaShopBundle/Controller/Admin/Sell/Address/AddressController.php

```diff
@@ -307,6 +307,10 @@ public function createAction(Request $request): Response
             if ($handlerResult->isSubmitted() && $handlerResult->isValid()) {
                 $this->addFlash('success', $this->trans('Successful creation.', 'Admin.Notifications.Success'));
 
+                if ($request->query->has('submitFormAjax')) {
```

Comment:
Why not using `$request->isXmlHttpRequest()`?

Reply:
It would be better, but I cannot guarantee all legacy calls using `submitFormAjax` parameter are proper `proper http request`.

In order to comply 100% with legacy behavior and ensure backward compatibility, I stay with previous implementation rules.

Reply:
:sob:

Reply:
I can add a `@todo PS 8.0.0` if you want 😅